components/permissions/contexts/brave_wallet_permission_context.cc

```diff
@@ -121,7 +121,9 @@ void BraveWalletPermissionContext::AcceptOrCancel(
     content::WebContents* web_contents) {
   PermissionRequestManager* manager =
       PermissionRequestManager::FromWebContents(web_contents);
-  DCHECK(manager);
+  if (!manager) {
+    return;
+  }
```

Comment:
There is no guarantee PermissionRequestManager(and whole set of typical tab helpers) is attached to web_contents